apps/ticket/src/app/(pages)/event/[eventId]/booking/[bookingId]/confirmed/_clientBoundary/EventInfoClient/index.tsx

```
@@ -0,0 +1,29 @@
+import classNames from "classnames/bind";
+
+import { Typography } from "@permit/design-system";
+
+import styles from "./index.module.scss";
+
+const cx = classNames.bind(styles);
+
+type Props = {};
+
+// TODO: params props 추가
+export const EventInfoClient = ({}: Props) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)